oscd-editor-sld.ts

```diff
@@ -143,7 +144,7 @@ export default class OscdEditorSld extends LitElement {
   gridSize = 32;
 
   @state()
-  nsp = 'esld';
+  nsp = 'esldoscd';
```

Comment:
I would probably stick with `esld` as a default namespace prefix, since the namespace URI is completely unambiguous. Is there a practical reason for changing it, or is this a theoretical precaution?